resource_management/.123env

```diff
@@ -0,0 +1,9 @@
+# This is a sample of what your .env file should look like if you wanted to run integration tests.
```

Comment:
Good catch. This file should not be here at all. As you mentioned we already have .env_smaple. By mistake it got committed.